pocs/camera/camera.py

```diff
@@ -496,14 +480,44 @@ def _process_fits(self, file_path, info):
         fits_utils.update_headers(file_path, info)
         return file_path
 
+    def _create_subcomponent(self, subcomponent, name, class_name, base_class):
```

Comment:
It's nice that this is split into a separate method as it is easier to read. Do you imagine it will be used more than once or by different subcomponents within one instance?

Reply:
Different subcomponents. The motivation for doing this was the introduction of filter wheel subcomponents in #769. I wanted to reuse the code for focuser subcomponent creation to create the filter wheel subcomponent so I needed to convert it into a generic method.